templates/jupyterhub-workspace.yaml

```diff
@@ -0,0 +1,307 @@
+kind: Template
```

Comment:
What is this?

Reply:
the old deployment template, I've created a new one to suit us better